### ISSUE TITLE
`frontmatter` package issue

### ISSUE THREAD

**Issue by sbaks0820** [bl4ck5un/ic3-website#272]
Looks like the latest pull of the repo doesn't make for me for some reason. @bl4ck5un have a look at this when you get a chance and see if you're able to reproduce this bug.

Steps:
1. Clone the repo and install all dependent python packages.
2. Run `make` and the first error encountered is:
```bash
processing ./content/events/devcon2.md
Traceback (most recent call last):
  File "main.py", line 286, in <module>
    index()
  File "main.py", line 43, in index
    events_toshow = event.get_event_list(e)
  File "/home/surya/projects/ic3-website/event.py", line 230, in get_event_list
    e.parse_md(env, f)
  File "/home/surya/projects/ic3-website/event.py", line 151, in parse_md
    post = frontmatter.load(md_src)
AttributeError: 'module' object has no attribute 'load'
Makefile:2: recipe for target 'site' failed
make: *** [site] Error 1
```

Perhaps it's a problem of the version of `frontmatter`.  My package manager installed version `frontmatter==3.0.5`.

**Comment by szcf-weiya** [comment]
I have met the same problem related to the `frontmatter` package, but not for this repository. I am not sure whether my solution works for you. 
In the beginning, it reports the same error, 
> AttributeError: 'module' object has no attribute 'load'

for `frontmatter.load()`, and I also installed `frontmatter==3.0.5`, but finally I found that there is another `frontmatter` package, https://pypi.org/project/python-frontmatter/, which is installed with

> pip install python-frontmatter

If I uninstall the `frontmatter==3.0.5` and install `python-frontmatter`, then the problem solved.
